src/ValueObject/Error/SystemError.php

```diff
@@ -83,7 +83,8 @@ public function getRectorClass(): ?string
     public function getRectorShortClass(): ?string
     {
         $rectorClass = $this->rectorClass;
-        if ($rectorClass !== null && $rectorClass !== '' && $rectorClass !== '0') {
+
+        if (!in_array($rectorClass, [null, ''], true)) {
```

Comment:
is `0` exclude by the rule? it was include `$rectorClass !== '0'`, if this map same mirror, this should be:

```suggestion
        if (!in_array($rectorClass, [null, '', '0'], true)) {
```